examples/run_nutation_amp_scopecapture.py

```diff
@@ -0,0 +1,111 @@
+"""
+Amp varied Nutation
```

Comment:
This is not an amplitude, very mutation, right?  This was adapted from an earlier example, which is as it should be, but the doc string at the top needs to be edited, and a `git mv` may be in order.